### PR TITLE
Layout cfg

### PR DESCRIPTION
**Pull request by cristianvasquez**
I refactored a little and added [layoutCfg](https://g6.antv.vision/en/docs/api/graphLayout/dagre) to be passed as prop
 
```json
  layout: {
    type: 'dagre',
    rankdir: 'LR', // The center of the graph by default
    align: 'DL',
    nodesep: 20,
    ranksep: 50,
    controlPoints: true,
  }
  ```

### CODE REVIEW

**Review by tpluscode — CHANGES_REQUESTED** [review]
Was it the intention to change code layout? Different order of attributes, `script` moved before `template` etc? It makes it hard to make sense of the actual changes

Would you rework the PR so that there are only relevant changes?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zazuko/vue-graph-layout",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (tpluscode) [inline package.json:3]: I think this should be reverted and only bumped when releasing new version

**Comment by cristianvasquez** [comment]
Thanks for the response.

I'll mark this branch as a draft and prepare a separate branch with the specific changes to the config.

Regarding script before template, I thought, from the documentation, that this was the convention

 https://vuejs.org/api/sfc-script-setup.html

**Comment by tpluscode** [comment]
Following conventions are good and I welcome such change. Just let's keep them separate :)

**Pull request closed without merging**